frontend/src/app/core/api.service.ts

```diff
@@ -160,42 +178,21 @@ export class ApiService {
         action: A,
         optimistic = false
     ) {
-        // TODO: throw if `response.success` is false
-        return this.optimisticActionHandler.proposeAction(action, optimistic);
-    }
-
-    /**
-     * Proposes an action to the server
-     */
-    private async sendAction(action: ExerciseAction) {
-        const response = await new Promise<SocketResponse>((resolve) => {
-            this.socket.emit('proposeAction', action, resolve);
-        });
-        if (!response.success) {
+        if (this.isTimeTraveling) {
             this.messageService.postError({
-                title: 'Fehler beim Senden der Aktion',
-                error: response.message,
+                title: 'Die Vergangenheit kann nicht bearbeitet werden',
+                body: 'Deaktiviere den Zeitreise Modus, um Änderungen vorzunehmen.',
```

Comment:
This feature is named `Aufzeichnung` everywhere else, I'd probably also use that name here.